core/src/test/scala/kanaloa/reactive/dispatcher/queue/WorkerSpec.scala

```diff
@@ -21,6 +21,17 @@ class TestBackend(delay: FiniteDuration = Duration.Zero)(implicit system: ActorS
   }
 }
 
+class ControlledBackend()(implicit system: ActorSystem) extends Backend {
```

Comment:
I introduced this, as I wanted complete control over when a Backend's Actor was created, so that I could directly control the flow of a test vs waiting on a timeout, which usually is fine, but can get finicky sometimes.